cmd/pending_publisher/main.go

```diff
@@ -160,8 +163,17 @@ func main() {
 	rhsFactory := reverse_hash.NewFactory(cfg.CredentialStatus.RHS.GetURL(), cl, common.HexToAddress(cfg.CredentialStatus.OnchainTreeStore.SupportedTreeStoreContract), reverse_hash.DefaultRHSTimeOut)
 	revocationStatusResolver := revocation_status.NewRevocationStatusResolver(cfg.CredentialStatus)
 
+	mediaTypeManager := services.NewMediaTypeManager(
+		map[iden3comm.ProtocolMessage][]string{
+			protocol.CredentialFetchRequestMessageType:  {string(packers.MediaTypeZKPMessage)},
+			protocol.RevocationStatusRequestMessageType: {"*"},
+		},
+		*cfg.MediaTypeManager.StrictMode,
+		cfg.MediaTypeManager.Disable,
```

Comment:
why is it disabled here? enabled, if even it's not used. Because if for some reason publisher will expose API, we will have a security issue.